package/yast2-packager.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Oct  6 08:01:38 UTC 2016 - lslezak@suse.cz
+
+- Disable autorefresh for new added local repositories (cd/dvd,
+  dir, hd (including USB flash), iso and file URL protocols)
```

Comment:
inconsistent naming, ISO should be big, HD also and probably also CD/DVD as USB and URL is also big ones